ts/session/apis/snode_api/signature/signatureShared.ts

```diff
@@ -0,0 +1,75 @@
+import { GroupPubkeyType, Uint8ArrayLen100, Uint8ArrayLen64 } from 'libsession_util_nodejs';
+import { isEmpty } from 'lodash';
+import { getSodiumRenderer } from '../../../crypto';
+import { PubKey } from '../../../types';
+import { StringUtils } from '../../../utils';
+import { fromUInt8ArrayToBase64 } from '../../../utils/String';
+import { NetworkTime } from '../../../../util/NetworkTime';
+
+export type SnodeSigParamsShared = {
+  namespace: number | null | 'all'; // 'all' can be used to clear all namespaces (during account deletion)
+  method: 'retrieve' | 'store' | 'delete_all';
+};
+
+export type SnodeSigParamsAdminGroup = SnodeSigParamsShared & {
+  groupPk: GroupPubkeyType;
+  privKey: Uint8ArrayLen64; // len 64
```

Comment:
not sure if we need the comments here? Seems redundant

Reply:
moved to docString, the `Uint8ArrayLen64` does show as Uint8Array on hover.